### PR TITLE
8.2.3 

### PR DESCRIPTION
**Pull request by KipK**
has to redo PR #18 to solve conflicts. Looks all good here

for the release, a Warning should display that wifi fw > 4.1.7 is mandatory, it won't unlock evse module if not. 
This should then be released only when the next wifi fw release is published.

### CODE REVIEW

**Comment by jeremypoulter** [comment]
@KipK can you also remove the 3ph env from the GitHub workflow?

**Comment by KipK** [comment]
https://github.com/OpenEVSE/open_evse/pull/19/commits/efb16c4425eee7a7254097f3b447aee997212a82  does this ain't it?

**Comment by jeremypoulter** [comment]
Hum... The check is still failing.... https://github.com/OpenEVSE/open_evse/actions/runs/4672244148/jobs/8348851478#step:7:1

**Comment by KipK** [comment]
Seems it still use the old one as it's not merged yet

**Comment by jeremypoulter** [comment]
Yeah I guess it is...

File: platformio.ini [hunk 1]
```diff
@@ -15,19 +15,20 @@ src_dir = firmware/open_evse
...
-version = 8.2.2
+version = 8.2.3
 build_flags =
   -D OEV6
   -D RELAY_PWM
   -D SHOW_DISABLED_TESTS
   -D AMMETER
   -D RAPI
   -D RAPI_SERIAL
-  -D RAPI_WF
```
[the remark this answers is not in the document]
Comment (KipK) [reply platformio.ini:26]: Enabled by default? That means anybody can unlock the evse or change the charging rules easily.( Rented BnB, hostel, public places, ... )

[the remark this answers is not in the document]
Comment (jeremypoulter) [reply platformio.ini:26]: Yeah, but the web portal can be password protected and at some point will allow the AP password to be changed

[the remark this answers is not in the document]
Comment (jeremypoulter) [reply platformio.ini:26]: But is needed as people regularly lock themselves out of the device...

[the remark this answers is not in the document]
Comment (KipK) [reply platformio.ini:26]: > AP password to be changed

yeah I was about to add this too with the new flags

**Pull request merged**